src/Login/ProtectedContent.jsx

```diff
@@ -215,6 +218,25 @@ class ProtectedContent extends React.Component {
     return newState;
   };
 
+  /**
+   * Display access contol message if userAccessToSite property set and user does not have sufficient privileges
+   */
+  checkSiteAccess = (initialState) => {
+    const newState = { ...initialState };
+    // check for setup property
+    if (userAccessToSite?.enabled && newState.user.authz) {
+      // check user authorization
+      if (!Object.keys(newState.user.authz).length > 0) {
```

Comment:
I feel this is a very specific use case. If there are any `all_users_policies` or `anonymous_policies` specified in user.yaml, then this won't work?
Maybe better considering adding a specific policy to those user's who should have access, and reject if the user logged in doesn't have that policy

Reply:
the current way works for this use case, additional features could easily be added if anyone else ever requests this functionality

Reply:
Yeah my point was there might be alternative solutions that is more generic if we consider this as a product-level feature instead of a project-level feature

Reply:
made this area a little more robust by adding userAccessIncludes option